Makefile

```diff
@@ -28,6 +28,13 @@ test:
 test-race:
 	go test -race ./...
 
+.PHONY: fmt
+fmt:
+	gofmt -s -w .
+	gofumpt -extra -w .
+	gci write . -s standard,default
```

Comment:
nice! btw, what is `gci`, and how is it installed?

Reply:
[`gci`](https://github.com/daixiang0/gci) is a tool that controls golang package import order and makes it always deterministic. It sorts the imports by putting standard library first, then the external dependencies, then project dependencies. It can be installed by running `go install github.com/daixiang0/gci@latest`. I guess this line slipped here by accent. Let me know if I should keep it or remove.

Reply:
makes sense to me too! fwiw, i love the addition of `make fmt`.